openml_OS/models/api/v1/Api_flow.php

```diff
@@ -65,12 +66,12 @@ function bootstrap($format, $segments, $request_type, $user_id) {
     }
 
     if (count($segments) == 1 && $segments[0] == 'tag' && $request_type == 'post') {
-      $this->entity_tag_untag('implementation', $this->input->post('flow_id'), $this->input->post('tag'), false, 'flow');
+      $this->flow_tag($this->input->post('flow_id'), $this->input->post('tag'));
```

Comment:
I really really prefer the entity tag untag functions, as they have the logic in one place

Reply:
I understand the mechanism